src/View/Helper/FormHelper.php

```diff
@@ -165,7 +165,9 @@ class FormHelper extends Helper
             'submitContainer' => '<div class="submit">{{content}}</div>',
             //Confirm javascript template for postLink()
             'confirmJs' => '{{confirm}}',
-        ]
+        ],
+        // set HTML5 validation message to custom required/empty messages
+        'autoSetCustomValidity' => false,
```

Comment:
I wish those would be class consts or sth to avoid magic string typos when using it :)

Reply:
I wish, too, that php had atoms.